lib/go-tc/deliveryservice_servers.go

```diff
@@ -189,21 +189,33 @@ type DSServerResponseV30 struct {
 
 // DSServerV4 contains information for a V4.x Delivery Service Server.
 type DSServerV4 struct {
+	DSServerV40
+	ASNs []int64 `json:"asns"`
+}
+
+// DSServerV40 contains information for a V4.0 Delivery Service Server.
+type DSServerV40 struct {
 	DSServerBaseV4
 	ServerInterfaces *[]ServerInterfaceInfoV40 `json:"interfaces" db:"interfaces"`
 }
 
 // DSServerResponseV40 is the type of a response from Traffic Ops to a request
 // for servers assigned to a Delivery Service - in API version 4.0.
 type DSServerResponseV40 struct {
+	Response []DSServerV40 `json:"response"`
+	Alerts
+}
+
+// for servers assigned to a Delivery Service - in API version 4.0.
```

Comment:
GoDoc comment should start with an optional article and the name of the documented symbol, and be a complete sentence.